SegmentEditorLocalThreshold/SegmentEditorLocalThresholdLib/SegmentEditorEffect.py

```diff
@@ -153,6 +153,7 @@ def setMRMLDefaults(self):
     self.scriptedEffect.setParameterDefault(FEATURE_SIZE_MM_PARAMETER_NAME, 3)
     self.scriptedEffect.setParameterDefault(SEGMENTATION_ALGORITHM_PARAMETER_NAME, SEGMENTATION_ALGORITHM_GROWCUT)
     self.scriptedEffect.setParameterDefault(HISTOGRAM_BRUSH_TYPE_PARAMETER_NAME, HISTOGRAM_BRUSH_TYPE_DRAW)
+    self.scriptedEffect.setParameterDefault("TURN_OFF_INTERACTIONS", str(False))
```

Comment:
This is a Boolean flag - to follow existing conventions, please use integer parameter with value 0 and 1.